src/PowerShellEditorServices/Services/Analysis/AnalysisService.cs

```diff
@@ -530,12 +534,12 @@ private void LogAvailablePssaFeatures()
                         { settingParameter, settingArgument },
                         // We ignore ParseErrors from PSSA because we already send them when we parse the file.
                         { "Severity", new [] { ScriptFileMarkerLevel.Error, ScriptFileMarkerLevel.Information, ScriptFileMarkerLevel.Warning }}
-                    });
+                    }).ConfigureAwait(false);
 
                 diagnosticRecords = result?.Output;
             }
 
-            _logger.LogDebug(String.Format("Found {0} violations", diagnosticRecords.Count()));
+            _logger.LogDebug(String.Format("Found {0} violations", diagnosticRecords.Length));
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['diagnosticRecords' is null on at least one execution path.](https://app.codacy.com/manual/TylerLeonhardt/PowerShellEditorServices/pullRequest?prid=4812399)

Reply:
Hmm odd